lib-integration-java/src/main/java/com/digitalasset/integration/internal/xml/XmlUtils.java

```diff
@@ -45,6 +46,8 @@ public static void writeTo(Node node, OutputStream outputStream) throws Transfor
     public static void writeTo(Node document, Writer writer) throws TransformerException {
         Transformer transformer = TRANSFORMER_FACTORY.newTransformer();
         transformer.setOutputProperty ( "encoding" , "UTF-8" ) ;
+        transformer.setOutputProperty(OutputKeys.INDENT, "yes");
+        transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
```

Comment:
I needed an XML writer that would do some passable attempt at pretty-printing. I don't know if this change is back-compatible, or going to bug anyone else. If we want, I can put this formatting behind a boolean flag or something.